cmd/storage/internal/app/app.go

```diff
@@ -12,10 +12,25 @@ var (
 type (
 	// App is an application interface.
 	App interface {
-		Save(stationID string, key string, value []byte) error
-		Load(stationID string, key string) ([]byte, error)
+		Save(hash string, key string, value []byte) error
+		Load(hash string, key string) ([]byte, error)
+		
 		Info() string
+		
+		GetServiceMoneyByHash(hash string) (int, error)
```

Comment:
Так же метод должен будет обновить время последнего вызова
```suggestion
		Ping(hash string) (int, error)
```